x-pack/plugins/security_solution_serverless/public/upselling/sections/endpoint_management/endpoint_agent_tamper_protection.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { EuiCard, EuiIcon } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import styled from '@emotion/styled';
+
+const BADGE_TEXT = i18n.translate(
+  'xpack.securitySolutionServerless.rules.endpointSecurity.agentTamperProtection.badgeText',
+  {
+    defaultMessage: 'Endpoint Complete',
+  }
+);
+const CARD_TITLE = i18n.translate(
+  'xpack.securitySolutionServerless.rules.endpointSecurity.agentTamperProtection.cardTitle',
+  {
+    defaultMessage: 'Do more with Security!',
+  }
+);
+const CARD_MESSAGE = i18n.translate(
+  'xpack.securitySolutionServerless.rules.endpointSecurity.agentTamperProtection.cardMessage',
+  {
+    defaultMessage: 'Upgrade your license to {productTypeRequired} to use Agent Tamper Protection.',
+    values: { productTypeRequired: BADGE_TEXT },
```

Comment:
NIT: the productTypeRequired could be parametrized instead of hardcoded, like this one:

https://github.com/elastic/kibana/blob/30c2abdc91f425655bd758fee3d1a91bfc8465d7/x-pack/plugins/security_solution_serverless/public/upselling/pages/threat_intelligence_paywall.tsx#L17

So, in case of a PLI change, we would not need to come here and update the component.